### PR TITLE
[WDB-420] Select Videos by date

### PR DESCRIPTION
**Pull request by AArlauskas**
Frontend functionality to select video cards by date

### CODE REVIEW

**Review by kibartas — COMMENTED** [review]
I love it

**Review by kibartas — APPROVED** [review]

File: frontend/src/containers/LibraryPage/LibraryPage.js [hunk 1]
```diff
@@ -173,6 +183,22 @@ class LibraryPage extends React.Component {
...
+      if (ids.every((id) => selectedCards.includes(id))) {
+        const newSelectedCards = selectedCards.filter(
+          (id) => !ids.includes(id),
+        );
+        this.setState({ selectedCards: newSelectedCards });
+      } else {
+        let newSelectedCards = selectedCards;
+        ids.forEach((id) => {
+          if (!selectedCards.includes(id))
+            newSelectedCards = [...newSelectedCards, id];
```
Comment (tmaciulis22) [inline frontend/src/containers/LibraryPage/LibraryPage.js]:
```suggestion
        const newSelectedCards = selectedCards.slice();
        ids.forEach((id) => {
          if (!selectedCards.includes(id))
            newSelectedCards.push(id);
```

Comment (tmaciulis22) [reply frontend/src/containers/LibraryPage/LibraryPage.js]: This way you would not create new array each time a new id is added

**Review by tmaciulis22 — APPROVED** [review]
Beautiful ❤️ Just minor performance improvement to look at

**Pull request merged**